src/adapters/appnexusAst.js

```diff
@@ -269,18 +269,23 @@ function AppnexusAstAdapter() {
   }
 
   function outstreamRender(bid) {
-    window.ANOutstreamVideo.renderAd({
-      tagId: bid.adResponse.tag_id,
-      sizes: [bid.getSize().split('x')],
-      targetId: bid.adUnitCode, // target div id to render video
-      uuid: bid.adResponse.uuid,
-      adResponse: bid.adResponse,
-      rendererOptions: bid.renderer.getConfig()
-    }, handleOutstreamRendererEvents.bind(bid));
+    // push to render queue because ANOutstreamVideo may not be loaded yet
+    bid.renderer.push(() => {
+      window.ANOutstreamVideo.renderAd({
+        tagId: bid.adResponse.tag_id,
+        sizes: [bid.getSize().split('x')],
+        targetId: bid.adUnitCode, // target div id to render video
+        uuid: bid.adResponse.uuid,
+        adResponse: bid.adResponse,
+        rendererOptions: bid.renderer.getConfig()
+      }, handleOutstreamRendererEvents.bind(bid));
+    });
   }
 
-  function onOutstreamRendererLoaded() {
-    // setup code for renderer, if any
+  function onOutstreamRendererLoaded(bid) {
+    // adload.loadScript has loaded ANOutstreamVideo
+    bid.renderer.loaded = true;
```

Comment:
Added a commit to do this. Bidders can override the callback by passing a function on the `callback` property given to `Renderer.install`. If that's unnecessary or other changes needed, will update